app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/components/ManageFormAccessDialog/ManageUsers.tsx

```diff
@@ -31,7 +31,12 @@ export const ManageUsers = () => {
     let valid = true;
 
     // User already has access
-    if (usersWithAccess.find((user) => user.email === email && !hasOwnProperty(user, "expired"))) {
+    if (
+      usersWithAccess.find(
+        (user) =>
+          user.email.toLowerCase() === email.toLowerCase() && !hasOwnProperty(user, "expired")
```

Comment:
Need to trim here?

Reply:
The email was being trimmed before being passed into the function, but nothing wrong with belt+suspenders.